app/controllers/application_controller.rb

```diff
@@ -15,8 +15,8 @@ def authorize!
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << :first_name << :last_name << :display_name
-    devise_parameter_sanitizer.for(:account_update) << :first_name << :last_name << :display_name
+    devise_parameter_sanitizer.for(:sign_up) << :first_name << :last_name << :display_name << :avatar
```

Comment:
I think you can put these in an array:

```ruby
devise_parameter_sanitizer.for(:sign_up) << [:first_name, :last_name, :display_name, :avatar]
```

I would try it out and see if it works. It's a bit cleaner.